include/belle.hh

```diff
@@ -1196,8 +1196,13 @@ private:
 
   private:
 
+#ifdef _MSC_VER
+    template <typename SELF_T, typename RES_T>
+    static constexpr void send(SELF_T self, RES_T&& res)
```

Comment:
If you don't mind, I'd prefer to use the template names `Self` and `Res`. I can make the change when merging if that is alright with you.

Reply:
I commited the changes!